docs/data/apis/api-providers.mdx

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 60
+title: Providers
+---
+
+:::info
+
+On August 1, 2024, the SDF truncated historical data on its Horizon instances to one year. This update allows us to optimize performance and ensure a streamlined experience for all users. We encourage you to explore third-party ecosystem providers of Horizon, which may provide a longer history retention window as well as other features.
```

Comment:
Suggest removing the mix of third party and first party statements, and keep it all focused on the topic rather than on us:
```suggestion
On August 1, 2024, the SDF truncated historical data on its Horizon instances to one year. This update allows optimizes the performance of the SDF hosted Horizon and ensures a streamlined experience for all users of the publicly available API. Consider third-party ecosystem providers of Horizon, which may provide a longer history retention window as well as other features.
```